dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetFieldType.java

```diff
@@ -438,7 +438,7 @@ public boolean isHasRequiredChildren() {
         return false;
     }
 
-    public boolean isHasParent() {
+    public boolean hasParent() {
```

Comment:
`Is` allows for field to be interpreted in xhtml as getter. So it probably breaks xhtml in some places (`#{dsft.hasParent}` will not work - it has to be changed to `#{dsft.hasParent()}`)

Reply:
I'll refactor it back then for clearance.